.github/workflows/plugin.yaml

```diff
@@ -24,3 +31,19 @@ jobs:
         compiler:
           - gcc
           - clang
+  release:
+    needs:
+      - test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: build
+        run: mkdir -p plz-out/package
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: tatskaari/release-action@master
+        with:
+          release-files: plz-out/package
```

Comment:
Do we actually have anything to release for this?

Reply:
There are no files to release, no - but it didn't seem to work if I didn't provide anything at all here so I'm just using an empty directory.